go/cmd/server/server_test.go

```diff
@@ -232,26 +234,23 @@ func TestGetOnlineFeaturesSqliteWithLogging(t *testing.T) {
 	for tr.Next() {
 		rec := tr.Record()
 		assert.NotNil(t, rec)
-		values, err := test.GetProtoFromRecord(rec)
+		actualValues, err := test.GetProtoFromRecord(rec)
 
 		assert.Nil(t, err)
-		assert.Equal(t, len(values)-1 /*request id column not counted*/, len(expectedLogValues))
-		// Need to iterate through and compare because certain values in types.RepeatedValues aren't accurately being compared.
-		for name, val := range values {
+		// Need to iterate through and compare because certain actualValues in types.RepeatedValues aren't accurately being compared.
+		for name, val := range expectedLogValues {
 			if name == "RequestId" {
 				// Ensure there are request ids for each entity.
-				assert.Equal(t, len(val.Val), len(response.Results[0].Values))
+				assert.Equal(t, len(val.Val), len(actualValues[name].Val))
 			} else {
-				assert.Equal(t, len(val.Val), len(expectedLogValues[name].Val))
+				assert.Equal(t, len(val.Val), len(actualValues[name].Val))
 				for idx, featureVal := range val.Val {
-					assert.Equal(t, featureVal.Val, expectedLogValues[name].Val[idx].Val)
+					assert.Equal(t, featureVal.Val, actualValues[name].Val[idx].Val)
 				}
 			}
 
 		}
 	}
-	err = test.CleanUpFile(logPath)
-	assert.Nil(t, err)
 }
```

Comment:
Do we no longer need to clean up the generated log file?

Reply:
Yep, it's written to temp directory, which will be cleaned up by test framework